teams/index.html

```diff
@@ -115,8 +115,7 @@ <h4 class="text-center">Jean-Marie Burel</h4>
                       <p>
                           Jean-Marie Burel joined the staff of the Swedlow lab
                           in 2003. Since then, he's been contributing to the
-                          development of OME, and spends most of his time
-                          working on the OMERO.insight project. He received is
+                          development of OME. He received is
```

Comment:
received _his_ ?

Reply:
typo has been there a many years
Thanks for spotting it